components/pages/data-dictionary/index.tsx

```diff
@@ -60,49 +58,63 @@ const rightButtonsStyle = css`
 	align-items: center;
 `;
 
+const displayData = (data: Dictionary[], filters: FilterOptions[], dictionaryIndex: number) => {
+	const currentDictionary = data?.[dictionaryIndex];
+	// If the filter is not active or we just have nothing to filter, return the original data
+	if (!filters?.length) {
+		return currentDictionary;
+	}
+	return {
+		...currentDictionary,
+		schemas: currentDictionary?.schemas?.map((schema: Schema) => ({
+			...schema,
+			fields: schema.fields.filter((field: any) => {
+				// we are going to filter via the constraints that are given
+				if (filters?.includes('Required')) {
+					return !field?.restrictions?.required === true;
+				}
+				if (filters?.includes('All Fields')) {
+					return true; // If All Fields is selected, we include all fields
+				}
+				return false;
```

Comment:
this can probably be more concise like so:

```suggestion
return filters?.includes('All Fields')
```

Any time you find yourself explicitly return a `true` or `false` after checking for a conditional (true or false), chances are you can just return the result...because the result itself will be the boolean.. true or false

If you think about it this block is saying something like 
```js
if(true) {return true}
// then if it's not true, it must be false
return false
```